spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb

```diff
@@ -54,46 +54,60 @@
   end
 
   describe '#generate', :vcr do
-    it 'includes the veterans name' do
-      expect(subject).to include 'Cat Marie Power, Jr.'
-    end
+    shared_examples 'includes introduction' do
+      it 'includes the veterans name' do
+        expect(subject).to include 'Cat Marie Power, Jr.'
+      end
 
-    it 'includes the veterans birthdate' do
-      expect(subject).to include 'DOB: 10-10-1968'
+      it 'includes the veterans birthdate' do
+        expect(subject).to include 'DOB: 10-10-1968'
+      end
     end
 
-    it 'includes the veterans blood pressure readings' do
-      expect(subject).to include 'Blood pressure: 115/87'
-    end
+    context 'when pdf is for hypertension' do
+      let(:disability_type) { :hypertension }
 
-    it 'includes the veterans medications' do
-      dosages = parsed_medications_data.map do |per|
-        next if per['dosageInstructions'].blank?
+      it_behaves_like 'includes introduction'
 
-        "Dosage instructions: #{per['dosageInstructions'].join('; ')}"
-      end.compact
+      it 'includes the veterans blood pressure readings' do
+        expect(subject).to include 'Blood pressure: 115/87'
+      end
 
-      expect(subject).to include(*dosages)
-    end
+      it 'includes the veterans medications' do
+        dosages = parsed_medications_data.map do |per|
+          next if per['dosageInstructions'].blank?
```

Comment:
I don't know what `per` means, so suggesting:
```suggestion
        dosages = parsed_medications_data.map do |med|
          next if med['dosageInstructions'].blank?
```